src/main/java/com/mitchej123/hodgepodge/config/TweaksConfig.java

```diff
@@ -128,6 +128,10 @@ public class TweaksConfig {
     @Config.DefaultInt(900)
     public static int autoSaveInterval;
 
+    @Config.Comment("Backports 1.20's 'pause-when-empty-seconds' server property")
+    @Config.DefaultBoolean(true)
```

Comment:
I think this should absolutely not be enabled by default, one of the biggest benefits of running a server especially in a pack like ours is being able to keep things running when you log off

Reply:
It is off by default since the default value of `pause-when-empty-seconds` is 0

Reply:
That is the only change i made from the modern implementation